src/@types/State.ts

```diff
@@ -49,3 +49,35 @@ export interface Standard {
   name: string
   average: number
 }
+
+export interface IssuedRetiredGraphData {
+  lastUpdated: string
+  countriesData: IssuedRetiredDataCountry[]
+}
+
+export interface IssuedRetiredDataCountry {
+  countryCode: string
+  timeRanges: {
+    year: number
+    month: number
+    issued: number
+    retired: number
+  }[]
+}
+
+export interface CreditsHistoryChartData {
+  id: string
+  data: { x: Date; y: number }[]
```

Comment:
I would prefer you to declare a new interface for sub-types, not inline declarations like this. Also `x` and `y` are too generic.